locust/runners.py

```diff
@@ -71,10 +73,13 @@ def on_request_failure(request_type, name, response_time, response_length, excep
         def on_hatch_complete(user_count):
             self.state = STATE_RUNNING
             if environment.reset_stats:
-                logger.info("Resetting stats\n")
+                logger.info("Resetting stats")
                 self.stats.reset_all()
         self.environment.events.hatch_complete.add_listener(on_hatch_complete)
-    
+
+        if self.environment.shape_class:
+            logger.debug("Starting with shape class")
```

Comment:
Looks like you are kind of logging this twice? (here and on line 286)

Reply:
True. I'll remove this one.